agcounts/pow2.py

```diff
@@ -1,14 +1,37 @@
 """Functions for dealing with pow2 sample rates."""
 
+import logging
+
 import numpy as np
 from scipy.signal import lfilter, lfilter_zi
 
+try:
+    from mne import filter
+except ImportError:
+    filter = None
+
+logger = logging.getLogger(__name__)
 
-def resample_to_30hz(data, sample_rate):
+
+def resample_to_30hz(data, sample_rate, interpolate: bool = False):
```

Comment:
Can we add here some docstring to explain the parameters?
It seems in both cases of if-else we interpolate (I guess the first kind is a direct interpolation and the other one is after upsampling to 256 Hz?)